src/Sarif.Converters/ToolFileConverterBase.cs

```diff
@@ -75,5 +75,54 @@ internal static string GetLogicalLocationName(string parentKey, string fullyQual
 
             return logicalName;
         }
+
+        protected static Run PersistResults(IResultLogWriter output, IList<Result> results, string toolName)
+        {
+            var run = new Run()
+            {
+                Tool = new Tool { Name = toolName },
+                ColumnKind = ColumnKind.Utf16CodeUnits
```

Comment:
>ColumnKind.Utf16CodeUnits [](start = 29, length = 25)

Since this occurs several times, maybe there's a good place for a global constant `ColumnKind DefaultColumnKind = ColumnKind.Utf16CodeUnits`.